load-balance/src/App.tsx

```diff
@@ -0,0 +1,22 @@
+
+import '@patternfly/patternfly/patternfly.css';
+import '@patternfly/patternfly/utilities/Accessibility/accessibility.css';
+import '@patternfly/patternfly/utilities/Display/display.css';
+import '@patternfly/patternfly/utilities/Flex/flex.css';
+import '@patternfly/patternfly/utilities/Sizing/sizing.css';
+import '@patternfly/patternfly/utilities/Spacing/spacing.css';
+import { LoadBalanceStep } from './LoadBalanceStep';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <p>
```

Comment:
[:nail_care:] Not a big deal, but could be possible to remove the unneeded tags? :)